webpack.config.js

```diff
@@ -1,124 +1,138 @@
-'use strict';
-const path = require('path');
-const webpack = require('webpack');
-const assert = require('assert');
-const {BundleAnalyzerPlugin} = require('webpack-bundle-analyzer');
+'use strict'
+const path = require('path')
+const webpack = require('webpack')
+const assert = require('assert')
+const {BundleAnalyzerPlugin} = require('webpack-bundle-analyzer')
 
 function getDefaultConfiguration() {
   return {
-  cache: true,
-  performance: { hints: false },
-  stats: 'errors-only',
-  externals: [{
-    'lodash': '_'
-  }],
-  entry: './dist/npm/index.js',
-  output: {
-    library: 'ripple',
-    path: path.join(__dirname, 'build/'),
-    filename: `ripple-lib.default.js`,
-  },
-  plugins: [
-    new webpack.NormalModuleReplacementPlugin(/^ws$/, './wswrapper'),
-    new webpack.NormalModuleReplacementPlugin(/^\.\/wallet$/, './wallet-web'),
-    new webpack.NormalModuleReplacementPlugin(/^.*setup-api$/, './setup-api-web'),
-    new webpack.ProvidePlugin({ process: 'process/browser' }),
-    new webpack.ProvidePlugin({ Buffer: ['buffer', 'Buffer'] })
-  ],
-  module: {
-    rules: []
-  },
-  resolve: {
-    extensions: ['.js', '.json'],
-    fallback: { 
-      "buffer": require.resolve("buffer/"),
-      "assert": require.resolve("assert/"),
-      "url": require.resolve("url/"),
-      "stream": require.resolve("stream-browserify"),
-      "crypto": require.resolve("crypto-browserify")
+    cache: true,
+    performance: {hints: false},
+    stats: 'errors-only',
+    externals: [
+      {
+        lodash: '_'
+      }
+    ],
+    entry: './dist/npm/index.js',
+    output: {
+      library: 'ripple',
+      path: path.join(__dirname, 'build/'),
+      filename: `ripple-lib.default.js`
+    },
+    plugins: [
+      new webpack.NormalModuleReplacementPlugin(/^ws$/, './wswrapper'),
+      new webpack.NormalModuleReplacementPlugin(/^\.\/wallet$/, './wallet-web'),
+      new webpack.NormalModuleReplacementPlugin(
+        /^.*setup-api$/,
+        './setup-api-web'
+      ),
+      new webpack.ProvidePlugin({process: 'process/browser'}),
+      new webpack.ProvidePlugin({Buffer: ['buffer', 'Buffer']})
+    ],
+    module: {
+      rules: []
+    },
+    resolve: {
+      extensions: ['.js', '.json'],
+      fallback: {
+        buffer: require.resolve('buffer/'),
+        assert: require.resolve('assert/'),
+        url: require.resolve('url/'),
+        stream: require.resolve('stream-browserify'),
+        crypto: require.resolve('crypto-browserify'),
+        https: require.resolve('https-browserify'),
```

Comment:
Is this the only change in this file? Could we revert/undo the formatting changes? I think it will be safer/cleaner to avoid formatting changes, and consider doing them in a later PR that *only* changes formatting.